src/utils/formatBalance.ts

```diff
@@ -30,3 +32,25 @@ export const formatNumber = (number: number, minPrecision = 2, maxPrecision = 2)
   }
   return number.toLocaleString(undefined, options)
 }
+
+/**
+ * Method to format the display of wei given an ethers.BigNumber object
+ * Note: does NOT round
+ */
+export const formatBigNumber = (number: ethers.BigNumber, displayDecimals = 18, decimals = 18) => {
+  const balance = formatUnits(number, decimals)
+  const [leftSide, rightSide] = balance.split('.')
+
+  let charCount = displayDecimals
+
+  if (leftSide.length > 0) {
+    charCount += leftSide.length
+  }
+
+  // Add 1 for the decimal point
+  if (rightSide.length > 0) {
+    charCount += 1
+  }
+
+  return balance.substr(0, charCount <= balance.length ? charCount : balance.length)
```

Comment:
We have two routes, given a 4 decimals display of `2.1`

1. Like `toFixed` where it pads the decimals e.g. `2.1` --> `2.1000`

2. Like `toLocaleString` where it uses the max amount of decimals e.g. `2.1` --> `2.1`

I opted for #2.

Reply:
Should we also plug in `.toLocaleString()` right here to add `,` between number units if its meant to be used for UI only?  e.g.`1,250,000.00` instead of `1250000.00`
It will need to be converted to number beforehand though, `.toLocaleString()` on string representation does nothing